styles.css

```diff
@@ -153,7 +153,7 @@ html {
     border: none;
     cursor: pointer;
     list-style-type: none;
-    font-size: 16px;
+    font-size: 12px;
   }
 
   .detail-btn {
```

Comment:
- [ ] As the previous code reviewer asked, please fix the spacing in all your buttons. Currently they are getting larger the more larger the screen is. Please use the fixed padding you used but make de width static.
 - Your design:
 - 
<img width="554" alt="image" src="https://user-images.githubusercontent.com/61919617/228813347-86045701-e1be-46c3-b239-4e77a1276be9.png">

 - Figma template:
 - 
<img width="410" alt="image" src="https://user-images.githubusercontent.com/61919617/228813491-06dc1262-e299-48f3-ace4-568fbf381b52.png">